config/jobs/kubernetes/sig-network/dualstack-e2e.yaml

```diff
@@ -203,3 +203,56 @@ periodics:
     testgrid-tab-name: dualstack-azure-e2e-master
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: "Dual-stack e2e tests on a Kubernetes cluster provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud"
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure-dualstack
+    decorate: true
+    decoration_config:
+      timeout: 2h
+    always_run: false
+    optional: true
+    path_alias: k8s.io/kubernetes
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201005-0b243c0-master
+        command:
+        - runner.sh
+        - kubetest
+        args:
+        # Generic e2e test args
+        - --test
+        - --up
+        - --down
+        - --build=quick
+        - --dump=$(ARTIFACTS)
+        # Azure-specific test args
+        - --provider=skeleton
+        - --deployment=aksengine
+        - --aksengine-agentpoolcount=2
+        - --aksengine-admin-username=azureuser
+        - --aksengine-creds=$(AZURE_CREDENTIALS)
+        - --aksengine-orchestratorRelease=1.19
+        - --aksengine-mastervmsize=Standard_DS2_v2
+        - --aksengine-agentvmsize=Standard_D4s_v3
+        - --aksengine-deploy-custom-k8s
+        - --aksengine-location=westus2
+        - --aksengine-public-key=$(AZURE_SSH_PUBLIC_KEY_FILE)
+        - --aksengine-template-url=https://gist.githubusercontent.com/aramase/21c3bdb44509c0f70f71389903d49c67/raw/1a9957626edc2b5c7913c741b9487fde123d0570/dualstack-prow.json
+        - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
+        # Specific test args
+        - --test_args=--ginkgo.focus=\[Feature:IPv6DualStackAlphaFeature\]
+        - --ginkgo-parallel=1
+        securityContext:
+          privileged: true
+    annotations:
+      testgrid-dashboards: sig-network-dualstack-azure-e2e, provider-azure-dualstack, provider-azure-presubmit
```

Comment:
I never remember if you have to add this to the  `config/testgrids/config.yaml` 
Please, check that it has address to notify errors, it is mandatory or that was the intention to avoid stale jobs

Reply:
Added 

```yaml
      testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
```